components/freertos/include/freertos/ringbuf.h

```diff
@@ -297,6 +297,18 @@ BaseType_t xRingbufferAddToQueueSetRead(RingbufHandle_t ringbuf, QueueSetHandle_
  */
 BaseType_t xRingbufferAddToQueueSetWrite(RingbufHandle_t ringbuf, QueueSetHandle_t xQueueSet);
 
+/**
+ * @brief Check if semaphore belongs to ringbuffer
+ *
+ * This API checks if Semaphore returned from xQueueSelectFromSet belongs to this ring buffer
```

Comment:
``This API checks if queue set member returned from xQueueSelectFromSet is the read semaphore of this ring buffer. If so, this indicates the ring buffer has items waiting to be read.``